src/components/ProductTours/TourCollapsible.tsx

```diff
@@ -73,16 +89,21 @@ const TourCollapsible: FC<Props> = (
         >
           <p className="small">{intl.formatMessage(messages.collapsibleIntro)}</p>
           <Stack gap={3} className="mb-3">
-            {QUICK_START_GUIDE_STEPS.map(step => (
-              <Step
-                key={step.title}
-                icon={step.icon}
-                title={step.title}
-                timeEstimate={step.timeEstimate}
-                targetId={step.targetId}
-                onTourSelect={onTourSelect}
-              />
-            ))}
+            {QUICK_START_GUIDE_STEPS.map(step => {
```

Comment:
We will want to filter out steps we aren't showing, otherwise we'll see a conspicuous gap in the flow list
![image](https://github.com/user-attachments/assets/66e21bee-1715-4457-ae76-4f506c112a5e)